website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -151,6 +152,31 @@ A run will be triggered in this workspace when changes are detected in any of th
 }
 ```
 
+A run will be triggered in this workspace when changes are detected:
+- in any folder tree which has the leaf folder named `networking` and the change was made in a file with the extension `tf`
+- in any file changed in the folder `/base`, no subfolders are included
+- in any file changed in the folder `/submodule` and all its subfolders
```

Comment:
That's the last folder on the path tree.

```
               root /
         ┌──────────────┐
         ▼              ▼
     networking       module
     ┌────          ┌──────────────┐
     ▼              ▼              ▼
  main.tf       submodule     networking
                ┌────              ────┐
                ▼                      ▼
            networking              main.tf
             ┌───
             ▼
          main.tf
```
In this example it will match all the `main.tf`s whenever they are located in networking folder, but the networking folder has to be the last folder. For example the following path is not matched because, even though we have networking on the path, it is not the last folder:
```
        root /
          │
          ▼
      networking
          │
          ▼
       submodule
          │
          ▼
        main.tf
```

Reply:
OHHHHHHH - okay I tried to update my suggestion to clarify this a tiny bit. Let me know what you think!

Reply:
4249075f623344eeeb379bd8b0eaa4198ad92515